tutorials/tts/Vits_Training.ipynb

```diff
@@ -0,0 +1,430 @@
+    "Now that we looked at the VITS model, let's see how to train a VITS Model\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "7rHG-LERrPRY"
+   },
+   "outputs": [],
+   "source": [
+    "# NeMo's training scripts are stored inside the examples/ folder. Let's grab the vits.py file\n",
+    "# as well as the vits.yaml file\n",
+    "!wget https://raw.githubusercontent.com/NVIDIA/NeMo/$BRANCH/examples/tts/vits.py\n",
+    "!(mkdir -p conf \\\n",
+    "  && cd conf \\\n",
+    "  && wget https://raw.githubusercontent.com/NVIDIA/NeMo/$BRANCH/examples/tts/conf/vits.yaml \\\n",
+    "  && cd ..)\n",
+    "\n",
+    "# additional files\n",
+    "!mkdir -p tts_dataset_files && cd tts_dataset_files \\\n",
+    "&& wget https://raw.githubusercontent.com/NVIDIA/NeMo/$BRANCH/scripts/tts_dataset_files/cmudict-0.7b_nv22.10 \\\n",
+    "&& wget https://raw.githubusercontent.com/NVIDIA/NeMo/$BRANCH/scripts/tts_dataset_files/heteronyms-052722 \\\n",
+    "&& cd ..\n",
+    "        "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "Upv_LxBIsC51"
+   },
+   "source": [
+    "Let's take a look at the vits.py file\n",
+    "\n",
+    "```python\n",
+    "import pytorch_lightning as pl\n",
+    "\n",
+    "from nemo.collections.tts.models.vits import VitsModel\n",
+    "from nemo.core.config import hydra_runner\n",
+    "from nemo.utils.exp_manager import exp_manager\n",
+    "\n",
+    "# hydra_runner is a thin NeMo wrapper around Hydra\n",
+    "# It looks for a config named vits.yaml inside the conf folder\n",
+    "# Hydra parses the yaml and returns it as a Omegaconf DictConfig\n",
+    "@hydra_runner(config_path=\"conf\", config_name=\"vits\")\n",
+    "def main(cfg):\n",
+    "    # Define the Lightning trainer\n",
+    "    trainer = pl.Trainer(replace_sampler_ddp=False, **cfg.trainer)\n",
+    "    # exp_manager is a NeMo construct that helps with logging and checkpointing\n",
+    "    exp_manager(trainer, cfg.get(\"exp_manager\", None))\n",
+    "    # Define the VITS model, this will construct the model\n",
+    "    model = VitsModel(cfg=cfg.model, trainer=trainer)\n",
+    "    # Let's add a few more callbacks\n",
+    "    trainer.callbacks.extend([pl.callbacks.LearningRateMonitor()])\n",
+    "    # Call lightning trainer's fit() to train the model\n",
+    "    trainer.fit(model)\n",
+    "\n",
+    "\n",
+    "if __name__ == '__main__':\n",
+    "    main()  # noqa pylint: disable=no-value-for-parameter\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "6nM-fZO-s75u"
+   },
+   "source": [
+    "Let's take a look at the yaml config\n",
+    "\n",
+    "```yaml\n",
+    "name: &name VITS\n",
+    "\n",
+    "train_dataset: ???\n",
+    "validation_datasets: ???\n",
+    "sup_data_path: null\n",
+    "sup_data_types: null\n",
+    "\n",
+    "phoneme_dict_path: \"scripts/tts_dataset_files/cmudict-0.7b_nv22.10\"\n",
```

Comment:
please correct the dict version: https://github.com/NVIDIA/NeMo/blob/main/scripts/tts_dataset_files/ipa_cmudict-0.7b_nv23.01.txt